src/main/resources/application.yaml

```diff
@@ -18,4 +18,10 @@ spring:
         dialect=org.hibernate.dialect.PostgreSQLDialect
   jackson:
     default-property-inclusion: NON_NULL
-  flyway.baseline-on-migrate: true
\ No newline at end of file
+  flyway.baseline-on-migrate: true
+
+logging:
```

Comment:
Maybe not all developers need this property. 
Maybe it would be better to add application-dev.yml and add to git ignore it.